src/bindings/python/tests/test_runtime/test_infer_request.py

```diff
@@ -349,7 +353,7 @@ def check_fill_inputs(request, inputs):
 
 def test_infer_mixed_keys(device):
     core = Core()
-    model = core.read_model(test_net_xml, test_net_bin)
+    model = get_relu_model()
     core.set_property(device, {"PERF_COUNT": "YES"})
     model = core.compile_model(model, device)
```

Comment:
```python
def test_infer_mixed_keys(device):
    compiled_model = get_compiled_model()  # anything that matches two inputs
    
    img = generate_image()
    # ...
```
How about this? Thre is no need to use `PERF_COUNT` and making it device specific.

Reply:
removed